### PR TITLE
Update the repo links

### PR DESCRIPTION
**Pull request by tomayac**
`s/ScrollToTextFragment/scroll-to-text-fragment/g`

***
<a href="https://pr-preview.s3.amazonaws.com/tomayac/scroll-to-text-fragment/pull/124.html" title="Last updated on Jun 25, 2020, 1:56 PM UTC (46af308)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/scroll-to-text-fragment/124/1708d3f...tomayac:46af308.html" title="Last updated on Jun 25, 2020, 1:56 PM UTC (46af308)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1474,9 +1474,9 @@ <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="cont
    <div data-fill-with="spec-metadata">
     <dl>
      <dt>This version:
-     <dd><a class="u-url" href="https://wicg.github.io/ScrollToTextFragment/">https://wicg.github.io/ScrollToTextFragment/</a>
+     <dd><a class="u-url" href="https://wicg.github.io/scroll-to-text-fragment/">https://wicg.github.io/scroll-to-text-fragment/</a>
      <dt>Issue Tracking:
-     <dd><a href="https://github.com/wicg/ScrollToTextFragment/issues/">GitHub</a>
+     <dd><a href="https://wicg.github.io/scroll-to-text-fragment/issues/">GitHub</a>
```
Comment (nickburris) [inline index.html:1479]: Did bikeshed generate this? This doesn't link to the issue tracker, it should be https://github.com/wicg/scroll-to-text-fragment/issues/

Comment (tomayac) [inline index.html:1479]:
```suggestion
     <dd><a href="https://github.com/wicg/scroll-to-text-fragment/issues/">GitHub</a>
```

Comment (tomayac) [reply index.html]: Ooops, I'm sorry, I've just corrected this.

**Comment by bokand** [comment]
Thanks Thomas!

**Pull request merged**